nemo/collections/asr/data/audio_to_text.py

```diff
@@ -478,8 +478,13 @@ def __init__(self, tokenizer):
                 else:
                     self.is_aggregate = False
                 self._tokenizer = tokenizer
-
             def __call__(self, *args):
+                if len(args) == 1 and isinstance(args[0], list):
```

Comment:
Yikes. I do not want a positional arg check that jumps to an assumption that you're using a aggregate tokenizer. 

Find better way to do this ? Maybe flags, maybe some other tricks? I'd really rather not do arglist checking and making any assumption in side a crucial shared data loader.

Reply:
Also, from typing import Iterable and check isinstance(x, Iterable) instead of list

Reply:
@bmwshop could you incorporate this suggestion from Som "from typing import Iterable and check isinstance(x, Iterable)"